src/common/components/file-issue-details-button.tsx

```diff
@@ -55,7 +55,13 @@ export class FileIssueDetailsButton extends React.Component<FileIssueDetailsButt
         const body = this.props.deps.issueDetailsTextGenerator.buildGithubText(data);
 
         const encodedIssue = `/new?title=${encodeURIComponent(title)}&body=${encodeURIComponent(body)}`;
-        return `${this.props.issueTrackerPath}${encodedIssue}`;
+        let baseUrl = this.props.issueTrackerPath;
```

Comment:
It looks like there aren't any tests for this change?

Reply:
I was surprise the new code was cover though. As I mention before maybe we don't even need this PR as we are removing the old experience